docs/configurations.md

```diff
@@ -0,0 +1,235 @@
+
+| Name                                              | Description                                                                                      | Default                 |
+|---------------------------------------------------|--------------------------------------------------------------------------------------------------|-------------------------|
+| `scalar.db.server.port`                           | Port number for ScalarDB Server.                                                                 | `60051`                 |
+| `scalar.db.server.prometheus_exporter_port`       | Prometheus exporter port. Prometheus exporter will not be started if a negative number is given. | `8080`                  |
+| `scalar.db.server.grpc.max_inbound_message_size`  | The maximum message size allowed to be received.                                                 | The gRPC default value. |
+| `scalar.db.server.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.                                             | The gRPC default value. |
+| `scalar.db.server.decommissioning_duration_secs`  | The decommissioning duration in seconds.                                                         | `30`                    |
+
+For details about ScalarDB Server, see [ScalarDB Server](scalardb-server.md).
+
+## Other ScalarDB configurations
+
+The following are additional configurations available for ScalarDB:
+
+| Name                                                             | Description                                                                                                                                                                                                       | Default              |
+|------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| `scalar.db.metadata.cache_expiration_time_secs`                  | ScalarDB has a metadata cache to reduce the number of requests to the database. This setting specifies the expiration time of the cache in seconds.                                                               | `-1` (no expiration) |
+| `scalar.db.active_transaction_management.expiration_time_millis` | ScalarDB maintains ongoing transactions, which can be resumed by using a transaction ID. This setting specifies the expiration time of this transaction management feature in milliseconds.                       | `-1` (no expiration) |
+| `scalar.db.default_namespace_name`                               | The given namespace name will be used by operations that do not already specify a namespace. If you would like to use this setting with ScalarDB Server, configure this setting in the client-side configuration. |                      |
```

Comment:
`scalar.db.default_namespace_name` do not exist for ScalarDB <= 3.8, so we can remove it.
```suggestion
```

Reply:
Got it. I've removed that part.